### PR TITLE
fix(styles): don't let sidebar crossover footer

### PR DESCRIPTION
**Pull request by pierreneter**
Close: #2815

### CODE REVIEW

**Comment by EugeneHlushko** [comment]
Did not work for me locally, you sure it solved it on your end?

File: src/components/Site/Site.scss [hunk 1]
```diff
@@ -11,5 +11,9 @@
 
   &__sidebar {
     flex: 0 0 280px;
+    &::after {
```
Comment (EugeneHlushko) [inline src/components/Site/Site.scss]:
```suggestion

    &::after {
```

**Comment by pierreneter** [comment]
Sure, didn't work, that's not the way to fix it. I'm confused, next commit is right thing....

**Comment by EugeneHlushko** [comment]
@montogeek wanna give it a shot?

**Comment by montogeek** [comment]
Thanks!

**Pull request merged**